.github/pull_request_template.md

```diff
@@ -11,3 +11,4 @@ Staging - <https://docs-mongodbcom-staging.corp.mongodb.com/drivers/docsworker-x
 - [ ] Did you run a spell-check?
 - [ ] Did you run a grammar-check?
 - [ ] Are all the links working?
+- [ ] Are the facets and meta tags accurate?
```

Comment:
S: instead of assuming that the tags are already there, maybe this question could be phrased as:
"Are there accurate facets and meta tags on the changed pages?"

Reply:
Also, perhaps this item could contain a link to the list of accepted values for the different categories

Reply:
I'll add a link, but omit "changed pages" since the existing checklist items apply to that context.